src/router/routes/ModelizerRoute.js

```diff
@@ -0,0 +1,34 @@
+import ModelizerPage from 'src/pages/ModelizerPage.vue';
+
+const ModelizerLongRoute = {
+  name: 'modelizer',
+  path: '/modelizer/:projectName/:viewType',
+  component: ModelizerPage,
+  beforeEnter: (to, _from, next) => {
+    const valideViewTypes = ['model', 'text'];
+    const projects = JSON.parse(window.localStorage.getItem('projects') || '{}');
```

Comment:
This should probably now use the composable Project's function instead of directly interfacing with the localStorage

Reply:
Call something like `if (!getProjectById(params.projectId)`

Reply:
good point !

Reply:
Done